terraform/modules/matrix-service/infra/sqs.tf

```diff
@@ -11,3 +11,24 @@ resource "aws_sqs_queue" "query_deadletter_queue" {
   name                      = "dcp-matrix-query-deadletter-queue-${var.deployment_stage}"
   message_retention_seconds = 1209600
 }
+
+resource "aws_sqs_queue" "notification_queue" {
+  name                      = "dcp-matrix-notification-queue-${var.deployment_stage}"
+//  Queue visibility timeout must be larger than (triggered lambda) function timeout
```

Comment:
Humor me - could you explain why this is?

Reply:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html is a great read